build.gradle.kts

```diff
@@ -744,7 +744,8 @@ tasks {
       if (!it.name.startsWith("catalog") &&
         !it.name.startsWith("authorization") &&
         !it.name.startsWith("client") && !it.name.startsWith("filesystem") && !it.name.startsWith("spark") && !it.name.startsWith("iceberg") && it.name != "trino-connector" &&
-        it.name != "integration-test" && it.name != "hive-metastore-common" && !it.name.startsWith("flink")
+        it.name != "integration-test" && it.name != "hive-metastore-common" && !it.name.startsWith("flink") &&
+        it.name != "hadoop-gcs-bundle" && it.name != "hadoop-s3-bundle"
```

Comment:
better renaming to "gcs-bundle" and "aws-bundle", like this, it may not only about storage, it also includes the security and other things.